Client/Components/ComparisonComponent.razor.cs

```diff
@@ -97,6 +97,10 @@ public string GetComparisonPercentage(float comparisonValue, float originValue)
 
         public float SetBarWidth()
         {
+            if(ComparisonCountryValue == 0 && OriginCountryValue == 0)
+            {
+                return 1;
+            }
```

Comment:
What does this do?

Reply:
A way to avoid SetBarWidth returning 0 since that will cause an error